index.html

```diff
@@ -54,7 +83,7 @@ <h1>Hey there.<br />I'm Mesfin</h1>
         <div class="text2">
           I can help you build a product , feature or website Look through some
           of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
+          you need coded, don’t hesitate to contact me.
         </div>
       </div>
```

Comment:
https://github.com/mesfin345/Portfolio/blob/1f9005cb50e09e8b258df7245fc1739c7d6f8061/index.html#L258-L309
- *** I see some of your about-me section is already on the main branch, I am commenting on this line just to make sure you got the comment. 
- [ ] Although you use padding for the `about-me section` It might not be always the right way to give space between elements, padding is mostly to gain space between the `border` and `inner contents. Instead, apply margins (top, bottom, left, right) defined in the design. This way you can separate your section from other sections. This is one of the main requirement you use margin accordingly.